sdks/python/apache_beam/transforms/core.py

```diff
@@ -1618,18 +1601,10 @@ def MapTuple(fn, *args, **kwargs):  # pylint: disable=invalid-name
 
       beam.MapTuple(lambda a, b, ...: ...)
 
-  is equivalent to Python 2
-
-      beam.Map(lambda (a, b, ...), ...: ...)
-
   In other words
 
       beam.MapTuple(fn)
 
-  is equivalent to
```

Comment:
It looks like this part was accidentally removed. Could you add this back?

@lazylynx @tvalentyn @robertwb

Reply:
@aaltay Ah, that's true. I'll fix this ASAP.